woocommerce/woocommerce-ios
# [Product description AI] Build celebration screen

Part of: #9934


## Description

Adds the code for the product description generation celebration screen. 

## Testing instructions
- You can check the UI by using SwiftUI preview `ProductDescriptionGenerationCelebrationView`. 
- No further testing needed as we are not using this view as of now. 

## Screenshots
<img width="696" alt="Screenshot 2023-06-13 at 12 35 39 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/524475/3fa048a4-2fbe-479b-a264-03b396dd716f">


---
